packages/components/src/List/utils/getAriaProps.ts

```diff
@@ -0,0 +1,42 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2021 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+
+import { ListItemProps } from '../types'
+
+export const getAriaProps = (props: ListItemProps) => {
+  const ariaProps = {}
+  const restProps = {}
+  Object.entries(props).forEach(([propKey, propValue]) =>
+    propKey.startsWith('aria-')
+      ? (ariaProps[propKey] = propValue)
+      : (restProps[propKey] = propValue)
+  )
+
+  return {
+    ariaProps,
+    restProps,
+  }
```

Comment:
Rather than returning an object return this as a tuple:

```suggestion
  return [
    ariaProps,
    restProps,
  ]
```